deploy-as-code/helm/environments/works-dev.yaml

```diff
@@ -144,6 +144,7 @@ contract-service:
   contract-search-max-limit: 50
   works-contract-service-code: "WORKS-CONTRACT"
   contract-workflow-business-service: "CONTRACT"
+  contract-workflow-revision-business-service: "WORKORDER-REVISION"

```

Comment:
@shubhang-eGov keep the nomenclature consistent. Use CONTRACT-REVISION or REVISED-CONTRACT.

Reply:
done